.bit/responses/2.1-Week 2 Step 1.md

```diff
@@ -1,5 +1,5 @@
 ---
-files: n/a
+files: week2/index.js
```

Comment:
@emsesc is this the right way to do this?

Reply:
No, delete that and add this to the `test.parseimage.js` file:

```js
const fs = require('fs') //get the methods in the fs package

//if you wanna add more files, just put a comma after the filename (array)
const commit_file = ['week2/index.js']

for(var i = 0; i < commit_file.length; i++) {
    var a = commit_file[i];
    fs.access(commit_file[i], err => {
        if (err) {
            throw new Error("You did not commit '" + a + "'")
        }
    })
}
```